src/components/TxData.js

```diff
@@ -277,6 +300,10 @@ class TxData extends React.Component {
     this.props.history.push('/wallet/');
   }
 
+  isAddressMine = (address) => {
+    return address in this.state.walletAddressesMap;
```

Comment:
I think this would return true even for entries that are not supposed to.

For example, for this [integration test scenario](https://github.com/HathorNetwork/hathor-wallet-lib/blob/a73a4634025c963ec97f13851d32774472c41681/__tests__/integration/hathorwallet_others.test.js#L240-L250), the walletAddressMap would be:
```js
{
  address1: true,
  address2: true,
  address3: true,
  'invalid-address': false,
}
```

This `isAddressMine` would return a _truthy_ response for a `invalid-address` parameter in this case, right?

Reply:
Thanks, fixed in f108d5b44a0cfb7c4ee80e45891c0a7ef6c3e149